virtcontainers/network.go

```diff
@@ -1464,17 +1417,91 @@ func createEndpoint(netInfo NetworkInfo, idx int, model NetInterworkingModel) (E
 	return endpoint, err
 }
 
-// network is the virtcontainers network interface.
-// Container network plugins are used to setup virtual network
-// between VM netns and the host network physical interface.
-type network interface {
-	// run runs a callback function in a specified network namespace.
-	run(networkNSPath string, cb func() error) error
+// Network is the virtcontainer network structure
+type Network struct {
+}
+
+func (n *Network) trace(ctx context.Context, name string) (opentracing.Span, context.Context) {
```

Comment:
@sameo This is code moved mostly unmodified from virtcontainers/default_network.go?

Where there any changes needed?

Reply:
No functional changes, no.